builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -67,7 +67,7 @@ func TestAccVSphereVirtualMachine_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"vsphere_virtual_machine.foo", "disk.#", "2"),
 					resource.TestCheckResourceAttr(
-						"vsphere_virtual_machine.foo", "disk.0.template", template),
+						"vsphere_virtual_machine.foo", "disk.3582676876.template", template),
```

Comment:
Can we not do two disks on the basic test??  My vision is that the basic test would be 'basic' ...  one disk 😄